pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -1009,7 +1009,6 @@ func TestUpdateNodeStatusWithLease(t *testing.T) {
 	assert.Len(t, actions, 9)
 	assert.IsType(t, core.GetActionImpl{}, actions[7])
 	assert.IsType(t, core.PatchActionImpl{}, actions[8])
-	patchAction = actions[8].(core.PatchActionImpl)
```

Comment:
SA4006: this value of `patchAction` is never used

Reply:
can still keep the type assertion

Reply:
https://github.com/kubernetes/kubernetes/blob/223addae4242ef754a5e185c401d8b95dd30cc83/pkg/kubelet/kubelet_node_status_test.go#L1011-L1012

Line 1011 has been tested for type?